src/api.js

```diff
@@ -112,6 +112,38 @@ function resolve(res, status, message) {
   return res.status(status).send(message);
 }
 
+/***
+ Helper function to look up repo owner in the Azure table to get installation ID,
+ and authenticate with GitHub using that ID.
+
+ Receives owner's name, the Azure table, and github object.
+ Returns either authenticated github object or null
+***/
+async function installationAuthenticate(owner, directory, github) {
```

Comment:
Let's call the table `OwnersDirectory`, like it's called elsewhere.  That makes it easier to find with a string search.